app/controllers/workarea/storefront/searches_controller.decorator

```diff
@@ -6,7 +6,7 @@ module Workarea
         view_model_options
       )
 
-      head 404 if @autocomplete.no_results?
+      @search = Storefront::SearchViewModel.new(@autocomplete.response, view_model_options)
```

Comment:
Can we add content to the autocomplete view model instead? Easier to test, keeps the number of view models at a minimum, less confusing in the view.